src/FakeItEasy.Analyzer/Diagnostic.nuspec

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>FakeItEasy.Analyzer</id>
+    <version>1.0.0.0</version>
+    <title>FakeItEasy.Analyzer</title>
+    <authors>Thomas Levesque, FakeItEasy contributors</authors>
+    <description>Provides diagnostic analyzers to warn about incorrect usage of FakeItEasy.</description>
+    <language>en-US</language>
+    <projectUrl>http://fakeiteasy.github.io/</projectUrl>
+    <iconUrl>http://fakeiteasy.github.io/img/fakeiteasy_logo_256_square_white.png</iconUrl>
+    <licenseUrl>https://raw.githubusercontent.com/FakeItEasy/FakeItEasy/master/License.txt</licenseUrl>
+    <copyright>Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)</copyright>
+    <tags>FakeItEasy, analyzers</tags>
+  </metadata>
+  <!-- The convention for analyzers is to put language agnostic dlls in analyzers\portable50 and language specific analyzers in either analyzers\portable50\cs or analyzers\portable50\vb -->
+  <files>

```

Comment:
I do not understand how this comment relates to the following `file` tag. is dotnet the new portable50?